src/Context/Initializer/SilverStripeAwareInitializer.php

```diff
@@ -184,4 +185,12 @@ public function setRegionMap($regionMap)
     {
         $this->regionMap = $regionMap;
     }
+
+    public function setI18nLocale(?string $i18nLocale)
+    {
+        $defaultLocale = 'en_US';
+        $locale = $i18nLocale ?: $defaultLocale;
+        i18n::config()->set('default_locale', $locale);
```

Comment:
Why are we changing the config as well?

Reply:
To be consistent with https://github.com/silverstripe/silverstripe-framework/pull/11570 mostly. I didn't test to see if only setting config, or only calling set_locate() was enough to resolve the behat tests, however I see no reason to not do this and it could well prevent future issues so I think we should set both.

Reply:
Have changed solution somewhat